platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/module/telemetry/MapEventFactory.java

```diff
@@ -35,8 +36,8 @@ static OfflineDownloadEndEvent buildOfflineDownloadCompleteEvent(PhoneState phon
                                                                    @FloatRange(from = MapboxConstants.MINIMUM_ZOOM,
                                                                      to = MapboxConstants.MAXIMUM_ZOOM) Double maxZoom,
                                                                    String styleURL,
-                                                                   Long sizeOfResourcesCompleted,
-                                                                   Long numberOfTilesCompleted,
+                                                                   int sizeOfResourcesCompleted,
+                                                                   int numberOfTilesCompleted,
```

Comment:
Small note that unboxing looks fine, but it should be `long` instead of an `int`. At least that's what the underlying `OfflineDownloadEndEvent` takes as a parameter.

Reply:
Ah sorry, you've changed those as well!

Reply:
Actually, let me reopen this. `OfflineRegionStatus` returns completed counts as `long`s. Was there another reason for changing the values to `int`s here?

Reply:
It will fail schema test. `long` is not included in type `number`
```
if (type.equalsIgnoreCase("int") || type.equalsIgnoreCase("integer")
      || type.equalsIgnoreCase("double") || type.equalsIgnoreCase("float")) {
      type = "number";
}
```

Reply:
Doesn't that require only updating the test matcher?
https://github.com/mapbox/mapbox-gl-native/blob/7b12272979a2b9fe17c4bd497508112a81a19c41/platform/android/MapboxGLAndroidSDK/src/test/java/com/mapbox/mapboxsdk/module/telemetry/SchemaTest.java#L191-L194

`long` should be a `number` type after gson serialization as well.

Reply:
So you think we should keep these two parameters as long? I will revert them.

Reply:
Thanks @Chaoba.